### PR TITLE
Custom map Terra

### PR DESCRIPTION
**Pull request by Tarsassum**
Retrying request after fixing issues in PR#1008

### CODE REVIEW

File: terra.lua [hunk 1]
```diff
@@ -0,0 +1,358 @@
+local b = require "map_gen.shared.builders"
```
Comment (grilledham) [inline terra.lua:1]: Duplicate file?

**Comment by grilledham** [comment]
Thanks for the PR, I see you have met travis 😄

**Comment by Tarsassum** [comment]
Replaced by PR#1011, a single commit, passing Travis. I'd rather close this mess.

**Comment by Tarsassum** [comment]
Replaced by PR#1011, a single commit, passing Travis. I'd rather close this mess.

**Pull request closed without merging**